README.md

```diff
@@ -7,9 +7,11 @@ This SDK makes it easy to create custom activity packs for [VertiGIS Studio Work
 
 ## Requirements
 
--   VertiGIS Studio Workflow 5.20 or later if running on-premises
 -   The latest LTS version of [Node.js](https://nodejs.org/en/download/)
 -   A code editor of your choice. We recommend [Visual Studio Code](https://code.visualstudio.com/)
+-   A viewer using VertiGIS Studio Workflow 5.31 or later ([VertiGIS Product Installer Downloads](https://apps.vertigisstudio.com/downloads))
+    -   VertiGIS Studio Web 5.22 or later
+    -   Workflow 5.31 widget for ArcGIS Experience Builder or later
```

Comment:
Can we add a note to this saying that if you are using an older viewer you need to use `v2` of the SDK?

Reply:
@rcooney Could this be opening a bit of a can of worms with the whole "supported versions" notion? I hoped this repo was simplified in that it only describes what is needed for this current version. Reviewing the history will show that this was added when changing from v2 to v3. Otherwise, would we then also need to say which version is the earliest for v2 etc?

Reply:
I'm not too fussed on where the information ends up, but as a user of the SDK, I want to see:
- That v3 has breaking changes
- What those breaking changes mean to me
- What to do if I don't meet the requirements of v3

The auto generated release notes like https://github.com/vertigis/vertigis-workflow-sdk/releases/tag/v2.0.0 are probably the best place for this information. These can be changed after the fact, so maybe we just do that.